src/main/java/uk/gov/hmcts/divorce/citizen/event/CitizenSubmitApplication.java

```diff
@@ -43,28 +53,43 @@ public void configure(ConfigBuilder<CaseData, State, UserRole> configBuilder) {
 
         log.info("Submit application about to start callback invoked");
 
-        final CaseData caseData = details.getData();
+        CaseData caseDataCopy = details.getData().toBuilder().build();
+        caseDataCopy.setApplicationFeeOrderSummary(solicitorSubmitApplicationService.getOrderSummary());
+        caseDataCopy.setPayments(singletonList(createPendingPayment()));
 
         log.info("Validating case data");
-        final List<String> validationErrors = AwaitingPayment.validate(caseData);
+        final List<String> validationErrors = AwaitingPayment.validate(caseDataCopy);
 
         if (!validationErrors.isEmpty()) {
             log.info("Validation errors: ");
-            for (String error:validationErrors) {
+            for (String error : validationErrors) {
                 log.info(error);
             }
 
             return AboutToStartOrSubmitResponse.<CaseData, State>builder()
-                .data(caseData)
+                .data(caseDataCopy)
                 .errors(validationErrors)
                 .state(Draft)
                 .build();
         }
         return AboutToStartOrSubmitResponse.<CaseData, State>builder()
-                .data(caseData)
-                .state(AwaitingPayment)
-                .build();
+            .data(caseDataCopy)
+            .state(AwaitingPayment)
+            .build();
     }
 
+    private ListValue<Payment> createPendingPayment() {
+        Payment payment = Payment
+            .builder()
+            .paymentAmount(55000)
```

Comment:
maybe pass in the OrderSummary to get the amount from that?

Reply:
yes you are right will update